.github/workflows/deploy.yml

```diff
@@ -21,3 +21,8 @@ jobs:
       - name: deploy to GitHub Pages
         id: deploy
         uses: ./.github/actions/deploy-to-github-pages
+      - name: Purge Cloudflare cache
+        uses: jakejarvis/cloudflare-purge-action@v0.3.0
+        env:
+          CLOUDFLARE_ZONE: ${{ secrets.CLOUDFLARE_ZONE }}
+          CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
```

Comment:
Sorry, couldn't review this earlier: can we also add an `if: secrest.CLOUDFLARE_TOKEN != ''` so that deployments in forks don't fail?

Reply:
Great idea, I pushed this out here: https://github.com/git/git-scm.com/pull/1895.